### PR TITLE
feat: create mod card template

### PR DESCRIPTION
**Pull request by DennaGrey**
# Pull Request
## Description:
Creating a template for the moderator's "cards" on the community page.
## Related Issue:

Closes #XXXXX

### CODE REVIEW

**Comment by nhcarrigan** [comment]
Remind me to review this in the morning. 💜

File: templates/moderator-card.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<style>
+  .container1{
+    width: 450px;
+    height: 300px;
+    background: linear-gradient(35deg, #511E6A, #9C138F, #B695CE);
+    border-radius: 15px;
```
Comment (nhcarrigan) [inline templates/moderator-card.html:6]:
```suggestion
    border-radius: 15px;
    padding-top: 3px;
    padding-bottom: 10px;
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
image

Couple of tiny tweaks to give some space at the top and bottom. Thoughts?

File: templates/moderator-card.html (hunk 1 above)
Comment (Skyhero-admin) [reply templates/moderator-card.html:6]: looks good 👍

Comment (DennaGrey) [reply templates/moderator-card.html:6]: I like the bottom, but i kind of intentionally wanted it to touch at the top.

Comment (nhcarrigan) [reply templates/moderator-card.html:6]: Works for me then 💜

**Review by nhcarrigan — APPROVED** [review]

**Pull request merged**